test/renaissance/auctions/auction_test.exs

```diff
@@ -0,0 +1,139 @@
+defmodule Renaissance.Test.AuctionTest do
+  use Renaissance.DataCase
+
+  require Ecto.Query
+  alias Renaissance.{Auction}
+
+  describe "user" do
+    test "requires title" do
+      {:ok, datetime, 0} = DateTime.from_iso8601("2019-04-21T12:00:00+0000")
+
+      changeset =
+        Auction.changeset(
+          %Auction{},
+          %{
```

Comment:
Could you extract out the base-line attrs into global so you can re-use and then change it as necessary?

Reply:
Resolved with 159a775e493da7136d042a4f76b690b4d8cb2aea